client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx

```diff
@@ -156,6 +160,28 @@ const InvestigationTable: React.FC = (): JSX.Element => {
         return (<Popper {...props} style={{ width: 350 }} placement='bottom-start' />)
     }
 
+    const RowTooltip = (props: React.PropsWithChildren<{creationDate: Date; startTime: Date;}>) => {
+        const {creationDate, startTime} = props;
+        const noDataMessage = 'אין מידע אודות תאריכים לחקירה זו';
```

Comment:
You can place hard coded constants outside of the component.

Reply:
moved comp. outside and fixed